integration_tests/features/data_conversion.feature

```diff
@@ -7,40 +7,40 @@ Feature: Testing data conversions
         When <script> <command> runs
         Then the output matches with the files in <output>
 
-    Examples: ABI
-        | source          | command                       | output           | script      | 
-        | abi/input/test1 | -r abi_l1b -w geotiff -vv -f  | abi/output/test1 | geo2grid.sh |
-        | abi/input/test2 | -r abi_l1b -w geotiff -vv -f  | abi/output/test2 | geo2grid.sh |
-        | abi/input/test3 | -r abi_l1b -w geotiff -vv -f  | abi/output/test3 | geo2grid.sh |
-        | abi/input/test4 | -r abi_l1b -w geotiff -vv -f  | abi/output/test4 | geo2grid.sh |
-
-    Examples: ACSPO
-        | source            | command                                | output             | script        |
-        | acspo/input/test1 | acspo gtiff -vv --grid-coverage=0.0 -f | acspo/output/test1 | polar2grid.sh |
-        | acspo/input/test2 | acspo gtiff -vv --grid-coverage=0.0 -f | acspo/output/test2 | polar2grid.sh |
-
-    Examples: AMSR2_L1B
-        | source            | command                                                    | output             | script        |
-        | amsr2/input/test1 | amsr2_l1b awips -vv -g 211e -p btemp_36.5h btemp_89.0av -f | amsr2/output/test1 | polar2grid.sh |
-
-    Examples: AVHRR
-        | source            | command             | output             | script        |
-        | avhrr/input/test1 | avhrr gtiff -vv -f  | avhrr/output/test1 | polar2grid.sh |                               
-  
-    Examples: MODIS
-        | source            | command                                                                                                                               | output             | script        |
-        | modis/input/test1 | crefl gtiff -vv --true-color --false-color --fornav-D 10 --grid-configs ${POLAR2GRID_HOME}/grid_configs/grid_example.conf -g miami -f | modis/output/test1 | polar2grid.sh |
-
+#    Examples: ABI
```

Comment:
Why are these all commented out?

Reply:
To save time; these tests take a long time to run, and I am working with pre and post build actions. It's temporary.